faiss/python/CMakeLists.txt

```diff
@@ -61,6 +61,10 @@ swig_add_library(swigfaiss
   SOURCES swigfaiss.swig
 )
 
+if(FAISS_OPT_LEVEL STREQUAL "avx2")
+  set_target_properties(swigfaiss PROPERTIES OUTPUT_NAME "swigfaiss_avx2")
```

Comment:
This looks like a nice solution. Did you test it?

Reply:
I did indeed. :)
That's also how I found out that `OUTPUT_NAME` doesn't affect `swigfaiss.py` (not sure if this should be considered a bug in the swig-interface of CMake, but in any case, it doesn't matter much here, as we're renaming the file already).

You can download an artefact from https://github.com/conda-forge/faiss-split-feedstock/pull/34 and test it yourself.